sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -828,14 +825,14 @@
           {
             "name": "id",
             "in": "path",
-            "description": "The ID of the device whose device twin is to be updated",
+            "description": "The unique identifier of the device whose twin is to be updated.",
             "required": true,
             "type": "string"
           },
           {
             "name": "deviceTwinInfo",
             "in": "body",
-            "description": "The twin object containing the tags and desired properties to be updated",
+            "description": "The twin object containing the tags and desired properties to be updated.",
```

Comment:
"Updated twin object"?

Reply:
It is the input that you would pass in for the "update" operation. It is not the updated twin.